zeus/tests/test_sample_survey_for_linked_election.yml

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
I don't know much about tests in python/django, but I feel like I should ask that.

Assuming we'll have more of them - can we put all test samples in a subdir(e.g. `zeus/tests/samples`)?

Reply:
Fair point, there's a lot of these `file()` calls throughout the codebase. Maybe a separate branch?

`file` is removed in Python 3 so we'll have to touch them anyway :)